index.js

```diff
@@ -1,9 +1,9 @@
 var express = require("express");
-var http = require("http");
 var app = express();
+var dao = require('./app/dao/dao');
```

Comment:
way to layer dao?

Reply:
Yes, I thought that we will use a reference to the dao instead of a reference to a specific db client.